policy/README.md

```diff
@@ -365,6 +384,130 @@ If a vehicle is matched with a rule, then it _will not_ be considered in the sub
+
+See [Policy Requierment Examples](/policy/examples/requirements.md) for how this can be implemented.
+
+An agency's [Requirements](#requirements) endpoint contains a number of distinct parts, namely [metadata](#requirement-metadata) and [MDS version](#requirement-mds-version) (with sub sections on applicable providers and relevant [MDS APIs](#requirement-mds-apis)). 
+
+```jsonc
+{
+  "metadata": {
+    // metadata fields per the "Requirement Metadata" section
+  },
+  "mds_versions" [ 
+    {
+      "version" : "[MDS VERSION NUMBER]",
+      "provider_ids": [
+        // provider id array
+      ],
+      "mds_apis": [
+        {
+          "api_name" : "[MDS API]": {
+            // MDS endpoints, urls, optional fields
+          }
+        },
+        // other MDS APIs per the "Requirement MDS APIs" section
+      ]
+    }, 
+    // other MDS versions per the "Requriement MDS Version" section
+  }
+}
+```
+[Top][toc]
+
+#### Requirement Metadata
+
+Contains metadata applicable to the agency and at the top of its [Requirement](#requirement) data feed in the `metadata` section. 
+
+| Name                         | Type            | Required / Optional | Description              |
+| ---------------------------- | --------------- | -------- | ----------------------------------- |
+| `mds_release`                | text            | Required | Release of MDS that the requirements data feed aligns to, based on official MDS releases. E.g. "1.2.0" |
+| `version`                    | integer         | Required | Version of this file. Increment 1 with each modification. E.g. "3" |
+| `last_updated`               | [timestamp][ts] | Required | When this file `version` was last updated. E.g. "1611958740" |
+| `max_update_frequency`       | integer         | Required | The expected maximum frequency with which this file could be updated. E.g. "P1D" |
+| `omf_review`                 | Enum            | Required | yes/no. Was this file reviewed by OMF Staff for accuracy? E.g. "yes" |
+| `omf_review_date`            | [timestamp][ts] | Optional | If `omf_review`, add timestamp. E.g. "1611958749" |
+| `agency_uuid`                | UUID            | Required | UUID of the agency this file applies to. Must come from [agencies.csv](/agencies.csv) file. E.g. "737a9c62-c0cb-4c93-be43-271d21b784b5" |
+| `agency_name`                | text            | Required | Name of the agency this file applies to. E.g. "Louisville Metro" |
+| `agency_time_zone`           | text            | Required | Timezone used for dates and times across all MDS endpoints. E.g. "America/New_York" |
+| `agency_currency`            | text            | Required | Currency used for all monetary values across all MDS endpoints. E.g. "USD" |
```

Comment:
Feels a bit superfluous since `currency` is specified in each policy, and I don't think it's used anywhere else.

Reply:
I think it's good for an agency to specify the currency it wants here at a high level so it's clear, in case elsewhere in the spec a value is left off.